pyhelm/repo.py

```diff
@@ -1,13 +1,22 @@
-import cStringIO
+try:
+    from io import StringIO
```

Comment:
Use `BytesIO` instead of `StringIO`. `StringIO` requires a unicode string, which `_get_from_repo` doesn't return.